### DISCUSSION TITLE
If you ever encountered this error 'ERR_OUT_OF_RANGE' while building your keycloakify project

### DISCUSSION THREAD

**Discussion by kasir-barati** [keycloakify/keycloakify#280] [General]
I was playing around and messing with [keycloakify-starter](https://github.com/keycloakify/keycloakify-starter/) and I said I do not need extra page 1 and 2. So I just deleted them and fix the code, but not `package.json` file. So I got a weird error but thankfully readable and with a bit of attention I realized that I did not do this inside `package.json` 
![image](https://user-images.githubusercontent.com/73785723/228104520-bed0e3f2-d621-4a4b-a1db-756ae3a9a313.png)

So after that everything resolved.

**Comment by garronej** [comment]
Thank you, I'll update the readme!

**Comment by kasir-barati** [comment]
Note, sometimes it also randomly throw this error at me and I just do not change anything and just run `npm build` command again and it works. I am not sure why but if you encountered this issue try these two solutions. I hope this can help someone

**Reply by garronej** [reply]
Should have been fixed in the latest release
